The styleguide is not accessible at /styleguide

## Expected Behavior
The fractal styleguide should be available at `/styleguide` such as https://revenuedata.doi.gov/styleguide

## Actual Behavior
https://revenuedata.doi.gov/styleguide returns a 404 error.

On Federalist, I see the following error for the step related to building the styleguide:
```
> @ build-styleguide /tmp/site_repo
> cd nrrd-design-system && fractal build && cd ..

Error: Cannot find module 'chokidar'
    at Function.Module._resolveFilename (module.js:538:15)
    at Function.Module._load (module.js:468:25)
    at Module.require (module.js:587:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/tmp/site_repo/node_modules/@frctl/fractal/src/core/mixins/source.js:5:18)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Module.require (module.js:587:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/tmp/site_repo/node_modules/@frctl/fractal/src/core/entities/source.js:12:16)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
```

--- a/package.json
+++ b/package.json
@@ -36,6 +36,7 @@
   "dependencies": {
     "@frctl/fractal": "^1.1.7",
     "async": "^0.9.0",
+    "chokidar": "^2.0.0",
     "copy-webpack-plugin": "^4.3.1",
     "css-loader": "^0.28.7",
     "custom-event": "^1.0.0",
@@ -60,7 +61,6 @@
     "webpack": "*"
   },
   "devDependencies": {
-    "chokidar": "^2.0.0",
     "colors": "^1.1.0",
     "data-expression": "^0.3.2",
     "epipebomb": "^0.1.1",

